### DISCUSSION TITLE
SVG Nft is not showing up

### DISCUSSION THREAD

**Discussion by masstrepidation** [Cyfrin/foundry-full-course-cu#1798] [General]
SVG Nft is not showing up, even though the code is exactly the same as Patricks. I have tested the JSON Metadata and Image path many times in browser and it does show up in browser but will not import into metamask. Also the description and other attributes are not showing up...

**Comment by EngrPips** [comment]
Hello @masstrepidation, I had a similar issue, and I could not resolve it, but someone tried resolving it and wrote a post on how they resolved it. I will share the post link so you can look and see if it will help. here https://www.linkedin.com/pulse/making-your-nft-collection-yevhen-hrytsenko-mjxif/. Note: They used an anvil chain, but you could try the same thing for sepolia if you run a sepolia chain.
